### PR TITLE
Pin path-to-regexp

### PR DESCRIPTION
**Pull request by GilbertCherrie**
Update sinon to ~19.0.2 and pinned path-to-regexp to 0.1.10 to ensure we are using either the latest version of path-to-regexp or a version with patched vulnerabilities.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -144,6 +144,8 @@
     "moment-timezone": "^0.5.41",
     "moment": "~2.29.4",
     "node-sass": "^8.0.0",
-    "nth-check": "^2.1.1"
+    "nth-check": "^2.1.1",
+    "express/path-to-regexp": "0.1.10",
+    "sinon": "~19.0.2"
```
Comment (Fryguy) [inline package.json:149]: Why do we need this resolution for sinon if we've updated the dependency?

File: package.json [hunk 2]
```diff
@@ -144,6 +144,8 @@
     "moment-timezone": "^0.5.41",
     "moment": "~2.29.4",
     "node-sass": "^8.0.0",
-    "nth-check": "^2.1.1"
+    "nth-check": "^2.1.1",
+    "express/path-to-regexp": "0.1.10",
```
Comment (Fryguy) [inline package.json:148]: Can you add this in alphabetically?

Comment (GilbertCherrie) [reply package.json]: Should be fixed now

File: package.json [hunk 3]
```diff
@@ -141,9 +141,11 @@
   "packageManager": "yarn@4.3.1",
   "resolutions": {
     "d3-color": "~3.1.0",
+    "express/path-to-regexp": "0.1.10",
```
Comment (Fryguy) [inline package.json:144]: Can we add a `~` here?  I don't want to get stuck on this specific version if there are further fixes.

**Comment by Fryguy** [comment]
I think you accidentaly removed the yarn.lock changes from this PR

**Comment by GilbertCherrie** [comment]
> I think you accidentaly removed the yarn.lock changes from this PR

Yeah, it should be fixed now

**Review by Fryguy — APPROVED** [review]

**Pull request merged**